docs/src/API.md

```diff
@@ -0,0 +1,36 @@
+# API Documentation
+
+## Neural DE Layer Functions
+
+- `NeuralODE(model, tspan, solver, args...; kwargs...)` defines a neural ODE
```

Comment:
it may be good to do these as docstrings.

Reply:
I'll keep that for another commit.